README.md

```diff
@@ -5,47 +5,134 @@
+Today’s solutions for AI app development require extensive custom coding and infrastructure plumbing. Tracking lineage and versions between and across data transformations, models, and deployment is cumbersome.
 
 ## 💾 Installation
 
 ```python
 %pip install pixeltable
 ```
+> [!NOTE]
+> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+
+## 💡 Getting Started
+Learn how to create tables, populate them with data, and enhance them with built-in or user-defined transformations and AI operations.
 
-To verify that it's working:
+| Topic | Notebook |
+|:--------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| 10-minute tour of Pixeltable    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/pixeltable-basics.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| User-Defined Functions (UDFs)    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/udfs-in-pixeltable.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Comparing Object Detection Models | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/object-detection-in-videos.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
 ```python
```

Comment:
Add `import pixeltable as pxt`

Reply:
I think it's distracting. This is to focus on the code itself. All these snippets are missing pip install and import statements. None of them are runnable in isolation.

Reply:
I wasn't suggesting adding pip install everywhere, but I'd advocate for at least putting it is just this one code snippet. It clarifies the meaning of `pxt`. This one actually *will* work on its own!

Reply:
Done.